docs/secure-connections/sauce-connect/setup-configuration/specialized-environments.md

```diff
@@ -15,6 +15,65 @@ import TabItem from '@theme/TabItem';
 * The name of your closest regional Sauce Labs Data Center (see the [SC CLI](/dev/cli/sauce-connect-proxy/#--region) and [Data Center Endpoints](/basics/data-center-endpoints/).
 
 
+## Sauce Connect Docker Container Setup
+
+As an alternative to downloading and installing the SC Client, you can leverage Docker containers to manage Sauce Connect Proxy tunnels. See [Using Docker Containers to Install/Run Sauce Connect](/secure-connections/sauce-connect/installation/#using-docker-containers-to-installrun-sauce-connect) for use cases.
+
+### Running the SC Docker Image
+
+1. Before running the container, you'll need to pull the Sauce Connect Proxy Docker Image from the Docker Hub. This will always pull the latest version of Sauce Connect Proxy.
+  ```bash
+  $ docker pull saucelabs/sauce-connect
+  ```
+   * Or - if you _do_ want to use a specific SC version - you can specify that as a tag. The example below pulls v4.7.0.
+   ```bash
+   $ docker pull saucelabs/sauce-connect:4.7.0
```

Comment:
can we change it to 4.7.1,  no real reason :)

Reply:
Hi @waggledans ! Since 4.7.1 is the current version, wouldn't you just use `$ docker pull saucelabs/sauce-connect`? That was why I had written 4.7.0.

Reply:
yeah that's OK, ur right